models/linkages/R/write_restart.LINKAGES.R

```diff
@@ -134,16 +134,21 @@ write_restart.LINKAGES <- function(outdir, runid, start.time, stop.time,
   }
   print(paste0("runid = ", runid))
   
+  # Create a new enviroment
+  linkages_env <- new.env()
   # load output
-  load(outfile)
+  load(outfile, envir = linkages_env)
   
-  ntrees <- ntrees.kill[, ncol(ntrees.kill), 1]  # number of trees
+  ntrees <- linkages_env$ntrees.kill[, ncol(linkages_env$ntrees.kill), 1]  # number of trees
   
   if(sum(ntrees)==0) {
     #reloads spin up if theres nothing in the output file
     print('No survivors. Reusing spinup.')
-    load(file.path(outdir, runid,list.files(file.path(outdir, runid))[grep(list.files(file.path(outdir, runid)),pattern='linkages')][1]))
-    ntrees <- ntrees.kill[, ncol(ntrees.kill), 1]  # number of trees
+    # new enviroment for spin up data 
+    linkages_env_spinup <- new.env()
+    spinup_file <- file.path(outdir, runid,list.files(file.path(outdir, runid))[grep(list.files(file.path(outdir, runid)),pattern='linkages')][1])
+    load(spinup_file, envir = linkages_env_spinup)
```

Comment:
Thanks for pulling the file path out to a variable for a nice readability improvement!